system/wymeditor/lang/en.js

```diff
@@ -28,6 +28,8 @@ WYMeditor.STRINGS.en = {
     Title:            'Title',
     Relationship:     'Relationship',
     Alternative_Text: 'Alternative text',
+    Attribution:      'Attribution',
+    Attribution_placeholder: 'cc: author (date)',
```

Comment:
I'm still unsure what "cc" means here. If it's intended to be "Creative Commons", then this might actualy be misleading, since just "CC" is not specific enough. I think we should just use "e.g." here, since it's intended as a suggestion? Or actually, maybe "license: author (date)" was the intention and is actually helpful, since it suggests to people that a license is actually needed. Let's try that and see what feedback we get.